src/test/kotlin/org/opensearch/replication/MultiClusterRestTestCase.kt

```diff
@@ -421,6 +422,13 @@ abstract class MultiClusterRestTestCase : OpenSearchTestCase() {
     }
 
     protected fun wipeIndicesFromCluster(testCluster: TestCluster) {
+
+        val followerStatsMap: Map<String,Any> = OpenSearchRestTestCase.entityAsMap(testCluster.lowLevelClient.performRequest(Request("GET","/_plugins/_replication/follower_stats")))
```

Comment:
@saikaranam-amazon Just a thought, should we also handle the exception cases(if stop called failed) here maybe? 
e.g. 
- Ensure that all replication related tasks are cancelled. 
- Cleaning up replication metadata as well, if possible.

Reply:
yes, we can target for replication tasks but 
any failure to remove the blocks are still applicable in this case (as we currently, we don't have clean way to do that) and with metadata present, we might attempt to clean the state eventually. 


Probably, we should prioritise the issue to cleanly exit _stop replication API call. Thoughts?

Reply:
Makes sense. In that case, at least we need to randomize the leader and follower index names in all tests to avoid any cascading failures incase we aren't able to do a clean `_stop`. Probably we can fast follow after this PR.